.github/workflows/build-pr.yml

```diff
@@ -11,54 +11,75 @@ env:
   docker_log_file_ee: docker-hazelcast-ee-test.log
 
 jobs:
-  build-pr:
+  prepare:
     runs-on: ubuntu-latest
-    name: Build with default JDK
+    name: Prepare environment
+    outputs:
+      HZ_VERSION_OSS: ${{ steps.get_oss_vars.outputs.HZ_VERSION_OSS }}
+      HZ_VERSION_EE: ${{ steps.get_ee_vars.outputs.HZ_VERSION_EE }}
+      HAZELCAST_EE_ZIP_URL: ${{ steps.get_ee_vars.outputs.HAZELCAST_EE_ZIP_URL }}
     steps:
       - name: Checkout Code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
+      - name: Install xmllint
```

Comment:
Nit - this should probably go back where it was (i.e. reject a build before you setup the env)

Reply:
Fixed: https://github.com/hazelcast/hazelcast-docker/pull/774/commits/231f22d2a3f58bf0e7de77e22b25888752126893